tests/test_arrow.py

```diff
@@ -182,6 +182,23 @@ def test_strptime(self):
             2013, 2, 3, 12, 30, 45, tzinfo=tz.gettz("Europe/Paris")
         )
 
+    def test_fromordinal(self):
+
+        timestamp = time.time()
```

Comment:
Just a heads up, we've had a few problems in the past with using this way of getting a timestamp for tests. I don't think it'll be an issue here but feel free to hardcode this if you like.

Reply:
Yes I think it will be better to hard code this, just in case. Thanks for pointing this out.